CMakeLists.txt

```diff
@@ -109,54 +113,81 @@ function(build_schrodinger_library TARGET)
   add_library(${TARGET} ${SOURCE_LIST})
   add_library(schrodinger::${TARGET} ALIAS ${TARGET})
   setup_target(${TARGET})
+  # Auto-generate compile definitions
+  string(TOUPPER ${TARGET} TARGET_UPPER)
+  target_compile_definitions(${TARGET} PRIVATE IN_${TARGET_UPPER}_DLL)
 endfunction()
 
-set(RDKIT_EXTENSIONS_LIB rdkit_extensions)
-build_schrodinger_library(${RDKIT_EXTENSIONS_LIB})
-target_compile_definitions(${RDKIT_EXTENSIONS_LIB}
-                           PRIVATE IN_RDKIT_EXTENSIONS_DLL)
+# schrodinger::rdkit_extensions library
+set(RDKIT_EXTENSIONS_TARGET rdkit_extensions)
+build_schrodinger_library(${RDKIT_EXTENSIONS_TARGET})
 target_link_libraries(
-  ${RDKIT_EXTENSIONS_LIB}
+  ${RDKIT_EXTENSIONS_TARGET}
   PRIVATE Boost::filesystem
           Boost::iostreams
           Boost::json
           Boost::serialization
-          RDKit::CIPLabeler
+          fmt::fmt
           RDKit::ChemReactions
           RDKit::DetermineBonds
           RDKit::DistGeomHelpers
           RDKit::MarvinParser
           RDKit::RDInchiLib
-          fmt::fmt
           SQLite::SQLite3
           ZLIB::ZLIB
           ${ZSTD_LIB_NAME})
 
-set(SKETCHER_LIB sketcher)
-build_schrodinger_library(${SKETCHER_LIB})
-target_compile_definitions(${SKETCHER_LIB} PRIVATE IN_SKETCHER_DLL)
-target_link_libraries(${SKETCHER_LIB} PRIVATE Qt6::Widgets Qt6::Svg
-                                              ${RDKIT_EXTENSIONS_LIB})
-# Enable Qt MOC and UIC for the sketcher library
-set_target_properties(${SKETCHER_LIB} PROPERTIES AUTOMOC ON)
-set_target_properties(${SKETCHER_LIB} PROPERTIES AUTOUIC ON)
-set(UI_DIR ${PROJECT_SOURCE_DIR}/src/schrodinger/${SKETCHER_LIB}/ui)
-set_target_properties(${SKETCHER_LIB} PROPERTIES AUTOUIC_SEARCH_PATHS ${UI_DIR})
+# schrodinger::sketcher library
+set(SKETCHER_TARGET sketcher)
+build_schrodinger_library(${SKETCHER_TARGET})
+target_link_libraries(
+  ${SKETCHER_TARGET}
+  PRIVATE Boost::boost
+          fmt::fmt
+          Qt6::SvgWidgets
```

Comment:
I think we still want `Qt6::Svg` here instead of `SvgWidgets` (see my comment above).